pkgs/development/python-modules/kestra/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  requests,
+}:
+buildPythonPackage rec {
+  pname = "kestra";
+  version = "0.20.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-j2ASwbleCsMrqhefhdBwXfp6JA+tBr1CX/nIKO4sWZI=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    requests
+  ];
+
+  pythonImportsCheck = [
+    "kestra"
+  ];
+
```

```suggestion
  nativeCheckInputs = [ pytestCheckHook ];
```
To run the unit tests.
If the tests are not present in the Pypi archive you may need to fetch from Github instead.

---

Quick question, it's my first test implementation for python package. The tests are bundled, but required `json` files are missing. Should I download them using curl in the `checkPhase`? Or is there a better method for this?